boards.txt

```diff
@@ -16504,7 +16905,7 @@ deneyapkartg.menu.PartitionScheme.default=Default 4MB with spiffs (1.2MB APP/1.5
 deneyapkartg.menu.PartitionScheme.default.build.partitions=default
 deneyapkartg.menu.PartitionScheme.defaultffat=Default 4MB with ffat (1.2MB APP/1.5MB FATFS)
 deneyapkartg.menu.PartitionScheme.defaultffat.build.partitions=default_ffat
-deneyapkartg.menu.PartitionScheme.default_8MB=8M with spiffs (3MB APP/1.5MB SPIFFS)
+deneyapkartg.menu.PartitionScheme.default_8MB=8M Flash (3MB APP/1.5MB FAT)
```

Comment:
As this is a standard partition scheme used on many boards, lets keep the naming the same please.